scandium-core/src/main/java/org/eclipse/californium/scandium/dtls/ClientHandshaker.java

```diff
@@ -338,8 +313,7 @@ protected DTLSFlight receivedHelloVerifyRequest(HelloVerifyRequest message) thro
 
 		DTLSFlight flight = new DTLSFlight(getSession());
 		flight.addMessage(wrapMessage(clientHello));
-
-		return flight;
+		recordLayer.sendFlight(flight);
 	}
```

Comment:
Yes, we do ... :astonished: : Sry, I meaned [here](https://github.com/eclipse/californium/pull/7/files#diff-5da9149ee37db8bcc492b03743f00dfeR224) in ServerHandshaker. 

(I review in eclipse and comment in github, this is confusing :stuck_out_tongue: )